Rakefile

```diff
@@ -1,20 +1,19 @@
 require 'rspec/core/rake_task'
 require 'rubygems/package_task'
+require 'mixlib/shellout/version'
 
-Dir[File.expand_path("../*gemspec", __FILE__)].reverse.each do |gemspec_path|
-  gemspec = eval(IO.read(gemspec_path))
+Dir[File.expand_path("../*gemspec", __FILE__)].each do |gemspec_path|
```

Comment:
Was unsure why it was needed - the commit log doesn't seem to mention needing to build this in any specific order.  This was mostly a "I wonder what would break" change and so far, it doesn't seem to break anything.